src/platform/auth.ts

```diff
@@ -140,11 +140,16 @@ export const generateNewJwtSecret = async (): Promise<string> => {
 	return base32.encode(key).toString();
 };
 
+export const checkSudoValidity = async (
+	req: Request | sbvrUtils.HookReq,
+): Promise<boolean> => {
+	const user = await getUser(req, false);
```

Comment:
Why?

Reply:
`getUser` is the most annoying API we have -- that flag whether it should throw -- and since you're exporting the function it won't be long before someone will want the equivalent of `getUser(req, true)`. You can also *possibly* omit the second `getUser` call just a few lines above.